lms/templates/dashboard.html

```diff
@@ -134,6 +134,12 @@ <h2 class="header-courses">${_("My Courses")}</h2>
         % endif
     </div>
 
+    <div id="progressive-profile"></div>
+
+    <%static:require_module_async module_name="common/js/vendor/progressive-profiles" class_name="a">
```

Comment:
Still gotta figure out if we need this `class_name` argument for anything - it works set to any value, but it can't be omitted.

Also I had some thoughts about bundling `edx-profiles` to an AMD library but not concatenating/minifying it, then seeing if it can be picked up here - that might give a little more debuggability in the consumer (and it would handle minification itself)